.circleci/src/jobs/notify_android.yml

```diff
@@ -24,5 +24,5 @@ steps:
       name: Create release notes
       command: echo "export RELEASE_NOTES=$(yarn --silent --cwd tools manage-metadata parse-release-notes --android --language en)" >> ${BASH_ENV}
   - notify:
-      success_message: <<^ parameters.production_delivery >>[Development] <</ parameters.production_delivery >>Integreat, Malte and Aschaffenburg ${NEW_VERSION_NAME} have been released successfully on Android!\n${RELEASE_NOTES}\n${ANDROID_ARTIFACT_URLS}
+      success_message: <<^ parameters.production_delivery >>[Beta] <</ parameters.production_delivery >>Integreat, Malte and Aschaffenburg ${NEW_VERSION_NAME} have been released successfully on Android!\n${RELEASE_NOTES}\n${ANDROID_ARTIFACT_URLS}
```

Comment:
Yes, I think beta makes more sense than development since it is more for testing by UI/UX, Martin and other teams and not for us developers. We mostly use webnext for development I guess. Also it is the naming that is used in the play store. Objections?

Reply:
No, I also think this makes sense.